src/NuGet.Core/NuGet.Configuration/PublicAPI.Unshipped.txt

```diff
@@ -1,28 +1,28 @@
-NuGet.Configuration.NamespaceItem
-NuGet.Configuration.NamespaceItem.Id.get -> string
-NuGet.Configuration.NamespaceItem.NamespaceItem(string id) -> void
-NuGet.Configuration.PackageNamespacesConfiguration
-NuGet.Configuration.PackageNamespacesConfiguration.AreNamespacesEnabled.get -> bool
-NuGet.Configuration.PackageNamespacesConfiguration.GetConfiguredPackageSources(string term) -> System.Collections.Generic.IReadOnlyList<string>
-NuGet.Configuration.PackageNamespacesSourceItem
-NuGet.Configuration.PackageNamespacesSourceItem.Namespaces.get -> System.Collections.Generic.IList<NuGet.Configuration.NamespaceItem>
-NuGet.Configuration.PackageNamespacesSourceItem.PackageNamespacesSourceItem(string name, System.Collections.Generic.IEnumerable<NuGet.Configuration.NamespaceItem> namespaceItems) -> void
-NuGet.Configuration.PackageNamespacesSourceItem.SetKey(string value) -> void
+NuGet.Configuration.PackagePatternItem
+NuGet.Configuration.PackagePatternItem.PackagePatternItem(string pattern) -> void
+NuGet.Configuration.PackagePatternItem.Pattern.get -> string
+NuGet.Configuration.PackageSourceMapping
+NuGet.Configuration.PackageSourceMapping.GetConfiguredPackageSources(string term) -> System.Collections.Generic.IReadOnlyList<string>
+NuGet.Configuration.PackageSourceMapping.IsEnabled.get -> bool
+NuGet.Configuration.PackageSourceMappingSourceItem
+NuGet.Configuration.PackageSourceMappingSourceItem.PackageSourceMappingSourceItem(string name, System.Collections.Generic.IEnumerable<NuGet.Configuration.PackagePatternItem> packagePatternItems) -> void
+NuGet.Configuration.PackageSourceMappingSourceItem.Patterns.get -> System.Collections.Generic.IList<NuGet.Configuration.PackagePatternItem>
```

Comment:
Does the order of patterns matter?  If not, then `ICollection<T>` might be a more minimalist interface for how the result is used.

Reply:
Fixed in the latest commit.

Reply:
I disagree with this.

Using too generic interfaces like that can lead to lots of accidental and unnecessary enumerations. 
We should stick with IList imo.

Reply:
reverted the last commit where I change the Patterns `type` to `ICollection`